packages/bot-web-ui/src/stores/quick-strategy-store.js

```diff
@@ -27,14 +29,22 @@ export default class QuickStrategyStore {
     @computed
     get initial_values() {
         return {
-            'quick-strategy__symbol': this.selected_symbol && this.selected_symbol.text,
-            'quick-strategy__trade-type': this.selected_trade_type && this.selected_trade_type.text,
-            'quick-strategy__duration-unit': this.selected_duration_unit && this.selected_duration_unit.text,
-            'quick-strategy__duration-value': this.input_duration_value,
-            'quick-strategy__stake': this.input_stake,
-            'quick-strategy__size': this.input_size,
-            'quick-strategy__loss': this.input_loss,
-            'quick-strategy__profit': this.input_profit,
+            'quick-strategy__symbol':
+                (this.qs_cache && this.qs_cache.selected_symbol && this.qs_cache.selected_symbol.text) ||
+                (this.selected_symbol && this.selected_symbol.text),
+            'quick-strategy__trade-type':
+                (this.qs_cache && this.qs_cache.selected_trade_type && this.qs_cache.selected_trade_type.text) ||
+                (this.selected_trade_type && this.selected_trade_type.text),
+            'quick-strategy__duration-unit':
+                (this.qs_cache && this.qs_cache.selected_duration_unit && this.qs_cache.selected_duration_unit.text) ||
+                (this.selected_duration_unit && this.selected_duration_unit.text),
+            'quick-strategy__duration-value': this.qs_cache.input_duration_value
+                ? this.qs_cache.input_duration_value
+                : this.input_duration_value,
+            'quick-strategy__stake': this.qs_cache.input_stake ? this.qs_cache.input_stake : this.input_stake,
+            'quick-strategy__size': this.qs_cache.input_size ? this.qs_cache.input_size : this.input_size,
+            'quick-strategy__loss': this.qs_cache.input_loss ? this.qs_cache.input_loss : this.input_loss,
+            'quick-strategy__profit': this.qs_cache.input_profit ? this.qs_cache.input_profit : this.input_profit,
```

Comment:
We can now use the null propagation operator in JS so we could refactor all of the highlighted code. Please see https://ponyfoo.com/articles/null-propagation-operator for more information. Example (I've provided an empty string as a default value, but play around with this):
```JS
{
    'quick-strategy__symbol': this.qs_cache?.selected_symbol?.text || this.selected_symbol?.text || '',
}
```